### DISCUSSION TITLE
PNPM workspace packages are not resolved by webpack

### DISCUSSION THREAD

**Discussion by dev-talha-akbar** [webpack/webpack#16569] [Q&A]
### Description
We're using PNPM as a package manager and using workspace feature from pnpm to resolve dependencies within the workspace. A simplified workspace for demonstration:
**Directory structure:**
```
- root
  - packages
    - app1
      - package.json
    - app2
      - package.json
  - package.json
```

**app1/package.json**
```
{
  name: 'app1',
  ...
}
```

**app2/package.json**
```
{
  name: 'app2',
  ...
  dependencies: {
    'app1': 'workspace:'*'
  },
  ...
}
```

### Expected Behavior
It works without any problems.

### Actual Behavior
`Module not found: Error: Can't resolve 'workspace:*' in 'path/to/file'`

**Comment by alexander-akait** [comment]
You should register an resolver for such cases, webpack is agnostic to package managers

**Reply by alexander-akait** [reply]
Please read https://github.com/pnpm/pnpm/issues/2255
